graphql/fetchUserProfileCard.ts

```diff
@@ -20,19 +20,19 @@ export type FetchUserProfileCardResponse = {
   users: Array<Users>;
 };
 export const FETCH_USER_PROFILE_CARD = gql`
-  query fetchUsers {
-    users(where: { enrolled: { _eq: true } }) {
-      id
-      name
-      link
-      profile_image
-      badges_earned
-      created_at
-      current_badge
-      coding_badge {
-        title
-        image
-      }
+query fetchUsers {
+  users(where: {enrolled: {_eq: true}}, order_by: {id: asc}) {
+    id
+    name
+    link
+    profile_image
+    badges_earned
+    created_at
+    current_badge
+    coding_badge {
```

Comment:
Can we just use the length of this array to determine the number of earned badges instead of making a new query?

Reply:
No, unfortuantely, that number is not accurate with the current foreign key.  I did keep the earnedBadges list as explained above, for now, but would like to look into implementing a foreign key.